source/_docs/z-wave/device-specific.markdown

```diff
@@ -614,4 +614,46 @@ switch:
             value_id: "{{ state_attr('sensor.scene_contrl_indicator','value_id') }}"
             value: "{{ states('sensor.scene_contrl_indicator')|int - 16 }}"
 ```
+
+### {% linkable_title HeatIt/ThermoFloor Z-Push Button 2/8 Wall Switch %}
+
+To get the Z-Push Button 2 or the Z-Push Button 8 working in Home Assistant, you must first edit the `COMMAND_CLASS_CENTRAL_SCENE` in your `zwcfg` file.
+
+1. Go the Z-Wave control panel in Home Assistant and make a note of the node number your wall switch has been assigned.
+2. *Stop* Home Assistant.
+3. Make a backup of your `zwfcg` file, just in case.
+4. In the `zwcfg` file, find the `Node id` that corresponds to the number you noted in the first step.
+5. Within the `Node id` you identified, highlight everything between `<CommandClass id="91"` and `</CommandClass>` (inclusive) and paste in the following:
+5.1. For the Z-Push Button 2:
+
+```xml
```

Comment:
In order not to break the Markdown numbering, you need to indent this code block at least 4 spaces relative to the list that it is in.